packages/go_router/lib/src/match.dart

```diff
@@ -4,16 +4,15 @@
 
 import 'package:flutter/foundation.dart';
 
-import 'go_route.dart';
-import 'go_route_information_parser.dart';
-import 'path_parser.dart';
-
-/// Each GoRouteMatch instance represents an instance of a GoRoute for a
-/// specific portion of a location.
-class GoRouteMatch {
-  /// Constructor for GoRouteMatch, each instance represents an instance of a
-  /// GoRoute for a specific portion of a location.
-  GoRouteMatch({
+import 'path_utils.dart';
+import 'route.dart';
+
+/// Each RouteMatch instance represents an instance of a GoRoute for a specific
+/// portion of a location.
+class RouteMatch {
```

Comment:
I am curious why we rename this class while most of the other classes still have GoRoute in it.

Reply:
This class isn't part of the public API, so we can safely rename without bumping the major version yet.

Reply:
my question was more of why we rename it.

It was probably my mistake to not make GoRouteMatch public because GoRouteInformationParser already expose GoRouteMatch as generic type

Reply:
I didn't see a reason to put `Go` in front of internal classes like RouteMatcher, RouteMatch, etc.

Reply:
sounds good